BREAKING_CHANGES.md

```diff
@@ -68,11 +69,11 @@ const response = await contract.LockToken(ctx, dto);
 
```

Comment:
In conjunction with below comment:

include `writes` vs `getWrites()` in the `BREAKING_CHANGES.md` example code

(github won't let me make a suggestion on line 67, maybe because its outside the modified block, so manually adding suggestion in this regular comment)

previous version sample 

```
const response = await contract.LockToken(ctx, dto);
```

becomes

```
const response = await contract.LockToken(ctx, dto);
expect(writes).toEqual(expectedWrites);
```